### PR TITLE
Edit on GitHub functionality tests

### LINKED ISSUE #965: [Parent: #851]Edit on GitHub functionality tests

### PR DESCRIPTION
**Pull request by HassanBahati**
#### Summary of Changes (What does this PR do?)
Tests for Edit on GitHub feature in documentation

- Verify Edit On GitHub Button is clickable
- Verify Destination of Edit On Github button

#### Status of maturity (all need to be checked before merging):

- [x] I've tested this locally
- [x] I consider this code done
- [x] This change ready to hit production in its current state
- [x] The title of the PR states what changed and the related issues number (used for the release note).
- [x] I've included issue number in the "Closes #ISSUE-NUMBER" part of the "[What are the relevant tickets?](#what-are-the-relevant-tickets)" section to [link the issue](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword).
- [x] I've updated corresponding documentation for the changes in this PR.
- [x] I have written unit and/or e2e tests for my change(s).

#### What are the relevant tickets?
#965 

- Closes #965 

#### Screenshots (optional)
Screenshot from 2022-11-27 11-42-40

### CODE REVIEW

File: docs/cypress.config.js [hunk 1]
```diff
@@ -5,5 +5,6 @@ module.exports = defineConfig({
     setupNodeEvents(on, config) {
       // implement node event listeners here
     },
+    chromeWebSecurity: false, //Skip Same Origin Policy to support tests that span multiple domains
```
Comment (HassanBahati) [inline docs/cypress.config.js:8]: Added `chromeWebSecurity: false` to the [cypress.config.js](https://github.com/airqo-platform/AirQo-frontend/blob/staging/docs/cypress.config.js) to skip Same Origin Policy to that blocks tests that span multiple domains .

 [References](https://reflect.run/articles/accessing-a-new-window-in-cypress-tests/)

**Review by 123MwanjeMike — APPROVED** [review]
Great stuff @HassanBahati 
LGTM

image

**Comment by Baalmart** [comment]
@HassanBahati , a few merge conflicts that need your attention to resolve.

**Comment by HassanBahati** [comment]
> @HassanBahati , a few merge conflicts that need your attention to resolve.

@Baalmart i have resolved them.

**Review by Baalmart — APPROVED** [review]
This LGTM, thanks @HassanBahati

**Pull request merged**